### ISSUE TITLE
Removing chart description does not happen with terraform apply

### ISSUE THREAD

**Issue by axeljonson** [splunk-terraform/terraform-provider-signalfx#235]
When I attempt to remove the 'description' argument from a chart and then run 
```
> terraform init
> terraform apply 
```
the description is not removed. I can see the change marked during the plan, but the change is not applied to the target deployment. The only way I was able to remove the description is to do a 
```
> terraform init
> terraform destroy
> terraform init
> terraform apply
```
Currently using:
```
> terraform -version
Terraform v0.12.26
+ provider.signalfx v4.23.2
```
Is there something else I need to do to make the 'description' argument be removed without doing the 'destroy'?
Thanks much!
Jon Axelson

**Comment by cory-signalfx** [comment]
I can reproduce this locally. Perhaps the API is ignoring a null description when it should be unsetting it? I'll do some testing!

**Comment by axeljonson** [comment]
Thanks @cory-signalfx !

**Comment by cory-signalfx** [comment]
No problem! I can also confirm that my guess above is true. I'll have a fix out soonish, I wanna verify some other resources to see if this is true also of dashboards, etc!

**Comment by cory-signalfx** [comment]
Hey @axeljonson, this is fixed up in [v4.26.1](https://github.com/splunk-terraform/terraform-provider-signalfx/releases/tag/v4.26.1). Thanks for reporting!

**Comment by axeljonson** [comment]
Thanks @cory-signalfx for the quick turn-a-round!!! I will move up to the new version!
